pytissueoptics/scene/logger/__init__.py

```diff
@@ -1 +1 @@
-from .logger import Logger
+from .logger import Logger, Segment
```

Comment:
This Segment import is not used.

Reply:
Resolved with 4408122614e7a8e265f5b5f95ee0644c2040a1cd